doc-src/SASS_REFERENCE.md

```diff
@@ -2534,6 +2534,53 @@ passed block are related to the other styles around where the block is defined.
       }
     }
 
+### Including a Mixin Dynamically
+
+There are times when it is convenient to call a mixin based on the
+result of a SassScript expression.
+
+For example, a library might want to delegate some styling to the
+application by allowing the application to register a mixin that the
+library would include where appropriate. Sometimes a content block is
```

Comment:
I'm the only person here, which feels dirty when reading need to create reference to function name through additional function (which makes some kinda wired recursion in design)? Why can't it be done in more natural way, like

``` Sass
@function test() { return 1 }
$testRef: test;

$output: $testRef()
```

Well, I understand that Sass has issue with determination of function due to string literals, but from other side if we will do this:

``` Sass
$testRef: test; // string literal

$output: $testRef()
```

It might behave same as Sass behaves now — since `test` it not a function, reference will not be received and we will get as output a plain `$testRef()` as string literal.

And if we want to get string literal, not reference, we can do this:

``` Sass
$testRef: test; // string literal

$output: $testRef
```

My only concern here that it won't be possible to properly pass `$testRef` reference to other vars, since we can't know does it hold function reference or variable.